tests/core/pyspec/eth2spec/utils/bls.py

```diff
@@ -78,7 +118,10 @@ def Verify(PK, message, signature):
 @only_with_bls(alt_return=True)
 def AggregateVerify(pubkeys, messages, signature):
     try:
-        result = bls.AggregateVerify(list(pubkeys), list(messages), signature)
+        if bls == arkworks_bls:  # no signature API in arkworks
+            result = py_ecc_bls.AggregateVerify(list(pubkeys), list(messages), signature)
```

Comment:
should we be using milagro here?

Reply:
Currently `arkworks_bls` basically means `py_ecc` (for signatures) and `arkworks` for KZG.

We can debate if we need the separate `milagro` and `arkworks` options, maybe we only need `py_ecc` and `fastest` ultimately.